sql-to-dbsp-compiler/SQL-compiler/src/test/java/org/dbsp/sqlCompiler/compiler/sql/postgres/PostgresInt2Tests.java

```diff
@@ -0,0 +1,397 @@
+                        "             (0.0::float8)," +
+                        "             (0.5::float8)," +
+                        "             (1.5::float8)," +
+                        "             (2.5::float8)) t(x);\n" +
+                        "  x   | int2_value \n" +
+                        "------+------------\n" +
+                        " -2.5 |         -2\n" +
+                        " -1.5 |         -2\n" +
+                        " -0.5 |          0\n" +
+                        "    0 |          0\n" +
+                        "  0.5 |          0\n" +
+                        "  1.5 |          2\n" +
+                        "  2.5 |          2"
+        );
+    }
+
+    // Fails because we round differently than Postgres
+    // R: (-2.5, -3)x1  L: (-2.5, -2)x1
+    // R: (-1.5, -2)x1  L: (-1.5, -1)x1
+    // R: (-0.5, -1)x1  L: (-0.5, 0)x1
+    // R: (0.5, 1)x1    L: (0.5, 0)x1
+    // R: (1.5, 2)x1    L: (1.5, 1)x1
+    // R: (2.5, 3)x1    L: (2.5, 2)x1
+    @Test @Ignore("fails because we round differently than Postgres")
+    public void testNumericRound() {
+        this.q(
+                "SELECT x, x::int2 AS int2_value " +
+                        "FROM (VALUES (-2.5::numeric)," +
+                        "             (-1.5::numeric)," +
+                        "             (-0.5::numeric)," +
+                        "             (0.0::numeric)," +
+                        "             (0.5::numeric)," +
+                        "             (1.5::numeric)," +
+                        "             (2.5::numeric)) t(x);\n" +
+                        "  x   | int2_value \n" +
+                        "------+------------\n" +
+                        " -2.5 |         -3\n" +
+                        " -1.5 |         -2\n" +
+                        " -0.5 |         -1\n" +
+                        "  0.0 |          0\n" +
+                        "  0.5 |          1\n" +
+                        "  1.5 |          2\n" +
+                        "  2.5 |          3"
+        );
+    }
+
+    @Test @Ignore("non decimals not suppported yet")
+    public void testNonDecimals() {
+        this.qs(
+                "SELECT '0b100101'::INT2 as x;\n" +
+                        " x \n" +
+                        "---\n" +
+                        " 37\n" +
+                        "(1 row)\n" +
+                        "\n" +
+                        "SELECT '0o273'::INT2;\n" +
+                        " x \n" +
+                        "---\n" +
+                        " 187\n" +
+                        "(1 row)\n" +
+                        "\n" +
+                        "SELECT '0x42F'::INT2;\n" +
+                        " x \n" +
+                        "---\n" +
+                        " 1071\n" +
+                        "(1 row)\n" +
+                        "\n"
+        );
+    }
+
+    @Test @Ignore("unimplemented operator")
+    public void testLeftShift() {
```

Comment:
Similar here, supporting `>>` using the Calcite grammar. Until someone adds support for it we should probably not add it ourselves.